docs/AuthRBAC.md

```diff
@@ -942,20 +942,20 @@ const authProvider = {
 const ProductEdit = () => (
     <Edit>
         <TabbedForm>
-            <TabbedForm.Tab label="Description" name="description">
+            <TabbedForm.Tab label="description">
```

Comment:
Please copy the exact doc and code from https://marmelab.com/ra-enterprise/modules/ra-rbac#tabbedform